pages/ar/cookbook/migrating-a-subgraph.mdx

```diff
@@ -2,6 +2,8 @@
 title: ترحيل Subgraph موجود إلى شبكة The Graph
 ---
 
+import { Video } from '@/components'
```

Comment:
I just did a quick test and it looks like we can make the component global (avoid having to import it every time we need it) by importing it in `MDXLayout.tsx` and including it in the `mdxComponents` object (with `Video: Video,` or just `Video,`). Could you please do that instead? And while you're at it, maybe do it with `<Difficulty />` too? 😇

Reply:
Yes, that works very well and have done it for `<Difficulty />` too. Thank you! ☺️